fmriprep/interfaces/images.py

```diff
@@ -205,71 +192,33 @@ def _run_interface(self, runtime):
+
         dropped_images = in_names[~valid]
+        self._results['t1w_dropped_list'] = dropped_images.tolist()
 
         # Ignore dropped images
         valid_fnames = in_names[valid]
-        valid_imgs = orig_imgs[valid]
-        reoriented = reoriented[valid]
+        self._results['t1w_valid_list'] = valid_fnames.tolist()
 
         # Set target shape information
         target_zooms = all_zooms[valid].min(axis=0)
         target_shape = all_shapes[valid].max(axis=0)
         target_span = target_shape * target_zooms
 
-        out_names = []
-        for img, orig, fname in zip(reoriented, valid_imgs, valid_fnames):
-            zooms = np.array(img.header.get_zooms()[:3])
-            shape = np.array(img.shape)
-
-            xyz_unit = img.header.get_xyzt_units()[0]
-            if xyz_unit == 'unknown':
-                # Common assumption; if we're wrong, unlikely to be the only thing that breaks
-                xyz_unit = 'mm'
-            # Set a 0.05mm threshold to performing rescaling
-            atol = {'meter': 5e-5, 'mm': 0.05, 'micron': 50}[xyz_unit]
-
-            # Rescale => change zooms
-            # Resize => update image dimensions
-            rescale = not np.allclose(zooms, target_zooms, atol=atol)
-            resize = not np.all(shape == target_shape)
-            if rescale or resize:
-                target_affine = np.eye(4, dtype=img.affine.dtype)
-                if rescale:
-                    scale_factor = target_zooms / zooms
-                    target_affine[:3, :3] = img.affine[:3, :3].dot(np.diag(scale_factor))
-                else:
-                    target_affine[:3, :3] = img.affine[:3, :3]
-
-                if resize:
-                    # The shift is applied after scaling.
-                    # Use a proportional shift to maintain relative position in dataset
-                    size_factor = target_span / (zooms * shape)
-                    # Use integer shifts to avoid unnecessary interpolation
-                    offset = (img.affine[:3, 3] * size_factor - img.affine[:3, 3]).astype(int)
-                    target_affine[:3, 3] = img.affine[:3, 3] + offset
-                else:
-                    target_affine[:3, 3] = img.affine[:3, 3]
-
-                data = nli.resample_img(img, target_affine, target_shape).get_data()
-                img = img.__class__(data, target_affine, img.header)
-
-            out_name = fname_presuffix(fname, suffix='_ras', newpath=runtime.cwd)
-
-            # Image may be reoriented, rescaled, and/or resized
-            if img is not orig:
-                img.to_filename(out_name)
-            else:
-                copyfile(fname, out_name, copy=True, use_hardlink=True)
-
-            out_names.append(out_name)
-
-        self._results['t1w_list'] = out_names
+        self._results['target_zooms'] = target_zooms.tolist()
+        self._results['target_shape'] = target_shape.astype(np.int64).tolist()
```

Comment:
Hehe indeed! I cast it because it was using floats, but I will double check it, just in case. If it is not possible to remove this casting, I will switch `np.int64` to `int`.